File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {
    let nums=nums1.slice()
    let median=0
    for(let i=0;i<nums2.length;i++){nums.splice(0,0,nums2[i])}
        nums.sort((a,b) => a-b)
        if(nums.length%2!==0){
            median=(nums.length-1)/2
            console.log(median)
            return nums[median]
        }
        if(nums.length%2==0){
            median=nums.length/2
            console.log(median)
            return ((nums[median]+nums[median-1])/2)
        }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] nums=nums1.Concat(nums2).ToArray();
        int median=0;
        double d=0.0;
        Array.Sort(nums);
        Console.WriteLine( string.Join(", ", nums));
        if(nums.Length%2!=0){
            median=(nums.Length-1)/2;
            return Convert.ToDouble(nums[median]);
        }
        if(nums.Length%2==0){
            median=nums.Length/2;
            return ((Convert.ToDouble(nums[median])+Convert.ToDouble(nums[median-1]))/2);
        }
        return d;
    }
}